env.example

```diff
@@ -14,3 +14,8 @@ CALNET_ENV=test
 CALNET_OIDC_CLIENT_ID=enter_the_id_here
 CALNET_OIDC_CLIENT_SECRET=enter_the_secret_here
 LANCEDB_URI=s3://mybucket
+# S3 configuration.
+BUCKET_NAME=willa
```

Comment:
does this do anything here, or are your changes to init-s3.sh above sufficient?

Reply:
I think it does make more sense to have the bucket name derived from `LANCEDB_URI`, so we can probably remove `BUCKET_NAME` as an env.  How we will do that in shell, I'm not as sure.